src/SincCoeffs.cc

```diff
@@ -146,7 +146,7 @@ class CircularAperture {
 };
 
 template <typename CoordT>
-class CircApPolar : public std::unary_function<CoordT, CoordT> {
+class CircApPolar {
```

Comment:
I couldn't find any uses of this code by doing a search, but I guess it doesn't hurt to leave it around?  (Same for `SincAperture` and `GaussPowerFunctor` below).